aaelf64/aaelf64.rst

```diff
@@ -1778,6 +1783,68 @@ The ``PAUTH`` and ``ENCD`` operators are defined in `PAUTHABIELF64`_.
   | 597        | \-         | R\_AARCH64\_AUTH\_TLSDESC\_ADD\_LO12   | G(ENCD(GTLSDESC(S)))                 | See `PAUTHABIELF64`_ |
   +------------+------------+----------------------------------------+--------------------------------------+----------------------+
 
+Relocations for Structure Protection Extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Structure Protection Extension defines a number of static
+relocations. The Structure Protection Extension is described in
+[STRUCTPROT_]. The Structure Protection Extension is in Alpha state.
```

Comment:
Wouldn't the more specific Deactivation Symbols RFC be more relevant to these relocations?

Reply:
There is one relocation for structure protection and one for deactivation symbols. I chose to concentrate on Structure Protection as that is the higher level description, and it includes a link to the deactivation symbols in it.

I could split the two apart a bit more and include links to both, but as these are LLVM discourse posts rather than formal descriptions I thought it best to stick to one.

Let me know if you want me to add some more context here? I could write a brief summary of structure protection, but could do that in a follow up patch.

Reply:
ok, fair enough.